joomla/joomla-cms
# [5.1] Convert mod_sampledata to service provider

### Summary of Changes

Converts the sample data administrator module to service provider.

### Testing Instructions

1. Make sure you have administrator module mod_sampledata published
2. You should see module output

![mod_sampledata](https://github.com/joomla/joomla-cms/assets/59166858/f90e8921-e2a1-4afa-bb4e-e71a5f1c45fa)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
